MSAL/src/native_auth/public/MSALNativeAuthUserAccountResult.swift

```diff
@@ -70,30 +72,54 @@ import Foundation
         }
     }
 
-    /// Retrieves an access token for the account.
+    /// Retrieves the access token for the default OIDC(openid, offline_access, profile) scopes from the cache.
     /// - Parameters:
     ///   - forceRefresh: Ignore any existing access token in the cache and force MSAL to get a new access token from the service.
     ///   - correlationId: Optional. UUID to correlate this request with the server for debugging.
     ///   - delegate: Delegate that receives callbacks for the Get Access Token flow.
-    @objc public func getAccessToken(forceRefresh: Bool = false, correlationId: UUID? = nil, delegate: CredentialsDelegate) {
-        Task {
-            let controllerResponse = await getAccessTokenInternal(
-                forceRefresh: forceRefresh,
-                correlationId: correlationId,
-                cacheAccessor: cacheAccessor
-            )
+    @objc public func getAccessToken(forceRefresh: Bool = false,
+                                     correlationId: UUID? = nil,
+                                     delegate: CredentialsDelegate) {
+        MSALLogger.log(
+            level: .info,
+            context: nil,
+            format: "Retrieving access token without scopes.")
 
-            let delegateDispatcher = CredentialsDelegateDispatcher(delegate: delegate, telemetryUpdate: controllerResponse.telemetryUpdate)
+        getAccessTokenInternal(forceRefresh: forceRefresh,
+                               scopes: [],
+                               correlationId: correlationId,
+                               delegate: delegate)
+    }
 
-            switch controllerResponse.result {
-            case .success(let accessTokenResult):
-                await delegateDispatcher.dispatchAccessTokenRetrieveCompleted(
-                    result: accessTokenResult,
-                    correlationId: controllerResponse.correlationId
-                )
-            case .failure(let error):
-                await delegate.onAccessTokenRetrieveError(error: error)
-            }
+    /// Retrieves the access token for the currently signed in account from the cache such that
```

Comment:
I would change the text (because I feel it's a bit convoluted):
`/// Retrieves the access token for the currently signed in account from the cache such that the scope of retrieved access token is a superset of requested scopes.`
into 
`/// Retrieves the access token for the currently signed in account from the cache which contains the requested scopes or a superset of those.`

Also what happens if there are 2 access tokens in the cache with the following scopes:
`'openid profile offline_access scope1'`
`'openid profile offline_access scope2'`
Retrieved in this order (so the one with `scope2` is last.
Which one gets retrieved? Why?
@nilo-ms @diegojerezba what do you guys think should happen in this scenario?

Reply:
If an external developer uses the "getAccessToken" function without any scopes, they do mind only receiving a valid access token that have default OIDC permission. In this case both cached tokens have this permission, so it's not important which one is returned.

About the suggested description: when a user specifies some scopes, it's not guaranteed that the access token returned will have all these scopes granted.

Reply:
I meant if they pass `openid profile offline_access` as scopes. Which one should they get.
Or let's say another case:
`openid profile offline_access scope1 scope2` stored
'openid profile offline_access scope1 scope3` stored
User retrieves `openid profile offline_access scope`. Which one should they get?

Regarding the `superset` description. Then maybe we shouldn't say they get back a superset, if that's not correct

Reply:
I can't say specifically what access token will be returned. I suppose none, because scope3 isn't linked to any access token stored.

Reply:
@spetrescu84 can we consider this conversation resolved?

Reply:
I tested this with requesting `openid profile offline_access scope1` and you get back `openid profile offline_access scope1 scope2 scope3` so there will be only one token all the time